CHANGES.md

```diff
@@ -2,6 +2,15 @@
 
 ### 1.99 - 2022-11-01
 
+#### Major Announcements :loudspeaker:
+
+- Starting with Cesium 1.100, CesiumJS will be published alongside two smaller packages `@cesium/engine` and `@cesium/widgets` [#10824](https://github.com/CesiumGS/cesium/pull/10824):
+  - The source code will been paritioned into two folders: `packages/engine` and `packages/widgets`.
+  - These workspaces packages will follow semantic versioning.
+  - These workspaces packages will be published as ES modules with TypeScript definitions.
+  - The combined CesiumJS release will continue to be published, however, only the `Assets`, `ThirdParty` and `Widgets` (with CSS files only) are available in the root level `Source` folder.
```

Comment:
as well as `Cesium.js` adn `Cesium.d.ts`, correct?

Reply:
That remains unchanged from the current release that's why I omitted it, but I can see that it can be confusing so I'll explicitly mention those too.